lib/matplotlib/tests/test_colors.py

```diff
@@ -156,6 +156,193 @@ def test_LogNorm():
     assert_array_equal(ln([1, 6]), [0, 1.0])
 
 
+def test_FuncNorm():
```

Comment:
However, the parametric test is not working, for some reason. Do you know, if apart from using the decorator, I have to do anything else? All other tests in the file are single method very basic tests (which is why I did it like that originally)